EnglishFilter/sections/foreign.txt

```diff
@@ -974,6 +974,9 @@ fofmyanmar.com##center > a[rel="noopener noreferrer"] > img
 ! https://github.com/AdguardTeam/AdguardFilters/issues/63770
 !+ NOT_OPTIMIZED
 sohu.com##a[href*="_Click_Flow_recommend_"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/74672
+sex1014004.pixnet.net#%#//scriptlet('abort-current-inline-script', 'window.leave')
+sex1014004.pixnet.net##a[href^="https://goo.gl/]
```

Comment:
better `pixnet.net##a[onmouseover^="MM_openBrWindow"]`. Same banner on many other subdomains.
Scriptlet also can be applied to the main domain.